rllib/connectors/common/numpy_to_tensor.py

```diff
@@ -103,17 +99,20 @@ def __call__(
             batch = {DEFAULT_MODULE_ID: batch}
 
         for module_id, module_data in batch.copy().items():
-            infos = module_data.pop(Columns.INFOS, None)
-            if rl_module.framework == "torch":
-                module_data = convert_to_torch_tensor(
-                    module_data, pin_memory=self._pin_memory, device=self._device
-                )
-            else:
-                raise ValueError(
-                    "`NumpyToTensor`does NOT support frameworks other than torch!"
-                )
-            if infos is not None:
-                module_data[Columns.INFOS] = infos
+            # If `rl_module` is None, leave data in numpy format.
```

Comment:
Oh, so the `rl_module` is not mandatory?

Reply:
Nope, it actually isn't. Only very few connectors actually need this information. Usually the ones checking, whether a module is stateful.